integration-tests/pom.xml

```diff
@@ -54,10 +54,15 @@
         <version>0.6.0-SNAPSHOT</version>
         <type>test-jar</type>
       </dependency>
+      <dependency>
+        <groupId>io.zipkin.zipkin2</groupId>
+        <artifactId>zipkin</artifactId>
+        <version>2.4.2</version>
```

Comment:
We don't need to specify the version here, the dependency pom can take care of it.

Reply:
This is inside the <dependencyManagement> section so the version is required. The integration-tests/pom.xml does not include the java-chassis-dependencies pom so a separate dependencyManagement is required.